code/modules/surgery/organs/organ.dm

```diff
@@ -81,7 +81,7 @@ var/list/organ_cache = list()
 		return
 
 	//Process infections
-	if ((status & ORGAN_ROBOT) || (sterile) ||(owner && owner.species && (owner.species.flags & IS_PLANT)))
+	if ((status & ORGAN_ROBOT) || sterile == 1 ||(owner && owner.species && (owner.species.flags & IS_PLANT)))
```

Comment:
The problem is not that sterile is being set - all the non-rotting organs I VV'd had it set to 0.